tsconfig.json

```diff
@@ -11,5 +11,7 @@
         "esModuleInterop": true,
         "skipLibCheck": true,
     },
-    "exclude": ["node_modules", "**/*.test.ts"]
-}
+    "exclude": [
+        "node_modules"
+    ]
```

Comment:
Could you ensure this doesn't affect the files outputted during `yarn build`?

Reply:
![image](https://github.com/atlowChemi/testim-junit-reporter/assets/109076747/ded14818-20d0-4049-bd6d-01a725d1138e)
run the one with the change and other without same sizes and also looked inside the index.js but didn't find anything